pause/pause.go

```diff
@@ -14,22 +13,37 @@
 // limitations under the License.
 //
 
-#include <signal.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <unistd.h>
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func sigHandler(sig os.Signal) {
+	var ret int
 
-static void sigdown(int signo) {
-	psignal(signo, "shutting down, got signal");
-	exit(0);
+	switch sig {
+	case os.Interrupt:
```

Comment:
You could maybe try to be consistent with the following statement by using `syscall.SIGINT`.